### ISSUE TITLE
What wallet do I use for miningpoolhub, multiwallets for multipool?

### ISSUE THREAD

**Issue by winnal** [aaronsace/MultiPoolMiner#477]
Sorry for all the questions.

What wallet do I use for miningpoolhub, and what if I want to use multi wallets since nicehash only accepts their own shares depositing to their wallet, and miningpoolhub don't seem to support internal wallets and I also can't figure out how to track stats and rates like nicehash provides and like dwarfpool/other pools

**Comment by winnal** [comment]
So as of now I literally am mining through miningpoolhub without receiving my shares... just for benchmarking and testing the switching-algo/pool...

**Comment by aaronsace** [comment]
Go to your MiningPoolHub account and set everything to auto exchange into one coin (i.e. Bitcoin) then add the one address (i.e. Bitcoin address) with a automatic withdraw amount (i.e. 0.1 BTC).

**Comment by winnal** [comment]
Ahh gotcha that makes sense, cheers.

**Comment by winnal** [comment]
I'm getting double the btc/d with miningpoolhub god bless!

**Comment by winnal** [comment]
I'm trying to mine VTC, using Lyra2rev2 but in my balances I'm getting some monacoin, is it just random because Lyra2rev2 will get both coins?

**Comment by fonyo** [comment]
You don't have to mess with the Hub workers stuff. That is what multipoolminer does for you.
MPM mines on the 17xxx ports which means if there are multiple coins supported by MPH for an algorithm, the more profitable coin will be mined. It is evaluated after a block is found so it is even better because you don't lose your shares due to early switching.

**Comment by winnal** [comment]
So it's best to leave it on no work or select the algorithm you're mining like lyra2rev2?

**Comment by aaronsace** [comment]
Just delete hub workers as you don't need them.

**Comment by imrecsoka** [comment]
If i look at MultiPoolMinerPlugins-v2.6.0.2c-Win i see many .bat files.... i just run the start.bat .... is this best practice. Also , not sure if this is a latest miner version "MultiPoolMinerPlugins-v2.6.0.2c-Win"
I see after the benchmarking which takes ages for my 3 GPUs, it starts mining musicoin i dont understand why because it is worth 1 cent.

**Comment by Diego32320** [comment]
Hi, I was wondering, does the wallet address used have to be a BTC wallet, or can it be the wallet we set to auto-exchange, (such as a Litecoin, or Bitcoin Cash)? Also, the auto-exchange only works for the multipoolhub pool, correct?

**Comment by scolarita13** [comment]
Hi, I have read all comments of this thread and something isn't clear.
I have set my multipoolminer to use all pools in this order: -poolname miningpoolhub,zpool,nicehash, hashrefinery. I have used the bitcoin address from my Quadriga account and set the miningpoolhub autoexchange to be paid in lite coin.
I have seen money are exchanged in lite coins. I checked the other pools (zpool,nicehash, hashrefinery) and all of them have balances in bitcoins. Where those balances go? Will they be autoexchange to lite coins eventually and transferred to miningpoolhub balance?
I asked other miners from my area.
Some of them say zpool,nicehash, and hashrefinery will pay you directly in your Quadriga bitcoin wallet.
Other miners say zpool,nicehash, and hashrefinery  will send those balances back to multipool hub when it reaches the pool threashold. Until then the balance in multipool hub will shows the balance for what you mined with algorithms of multipoolhub only.
What is the real answer? Maybe aaronsace can clarify.
Thank you,
scolaritaMiner
